carta/session.py

```diff
@@ -368,11 +368,15 @@ def open_image(self, path, hdu=""):
             The path to the image file, either relative to the session's current directory or an absolute path relative to the CARTA backend's root directory.
         hdu : {1}
             The HDU to select inside the file.
+        complex : {2}
+            Whether the image is complex. Set to ``False`` by default.
+        expression : {3}
+            Arithmetic expression to use if opening a complex-valued image. By default, the amplitude will be shown if the image is complex.
         """
-        return Image.new(self, path, hdu, False)
+        return Image.new(self, path, hdu, False, complex, expression)
 
-    @validate(String(), String(r"\d*"))
-    def append_image(self, path, hdu=""):
+    @validate(String(), String(r"\d*"), Boolean(), NoneOr(Constant(ArithmeticExpression)))
+    def append_image(self, path, hdu="", complex=False, expression=None):
```

Comment:
As above, the default should be set to amplitude here, and the description and validation should be updated.

Reply:
Done in f651be6.